spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
Ok, here's what I think we need to do, based on the feedback: reinstate `DURABLE` as a global property, make changes to the current validation process to do validation on values as well (not only on key presence), so binders can declare options they only support. Also, I think we need a mechanism at the global level that allows to detect what properties have been accessed by a binder and which ones weren't, and act accordingly - e.g. if a binder ignores a mandatory property in its implementation.